cbmc_viewer/sourcet.py

```diff
@@ -34,15 +34,15 @@ class Sources(enum.Enum):
 ################################################################
 # Source validator
 
-VALID_SOURCE = voluptuous.schema_builder.Schema({
+VALID_SOURCE = voluptuous.Schema({
     # Absolute path to the source root
-    voluptuous.schema_builder.Required('root'): str,
+    voluptuous.Required('root'): str,
```

Comment:
Required is already true for the whole schema, this misleading IMHO. Ironically, I think only the `Optional` on line 45 is 'required' here.